src/main/webapp/WEB-INF/pages/productPriceHistory.jsp

```diff
@@ -0,0 +1,37 @@
+<%@ page contentType="text/html;charset=UTF-8" %>
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+<%@ taglib prefix="fmt" uri="http://java.sun.com/jsp/jstl/fmt" %>
+<%@ taglib prefix="tags" tagdir="/WEB-INF/tags" %>
+
+<jsp:useBean id="product" type="com.es.phoneshop.model.product.Product" scope="request"/>
+<tags:master pageTitle="Product Details">
+  <p>
+    ${product.description}
+  </p>
+  <table>
+      <tr>
+        <td>Image</td>
+        <td><img src="${product.imageUrl}"></td>
+      </tr>
+      <tr>
+        <td>Price History:</td>
+        <td>
+          <h4 style="padding-left:2em">
+            <span style="display:inline-block; width:5em;">Start date:</span>
+            <span style="display:inline-block; width:5em;">Price:</span>
```

Comment:
Please move inline styles to CSS files. This way you can avoid code duplication.